src/Features/Core/Portable/Completion/CompletionHelper.cs

```diff
@@ -250,12 +250,6 @@ private int CompareMatches(PatternMatch match1, PatternMatch match2, CompletionI
                 return diff;
             }
 
-            var preselectionDiff = ComparePreselection(item1, item2);
```

Comment:
Looks like remove this check caused `VisualBasicCompletionCommandHandlerTests.BooleanPreselection2` to fail. After reading thru the comments in https://github.com/dotnet/roslyn/issues/4892, I think we need to preserve the behavior in VB that type `f` and `t` would preselect `False` and `True`.